### DISCUSSION TITLE
Remove Share-Button in Embedded-Videos

### DISCUSSION THREAD

**Discussion by Husky110** [mediacms-io/mediacms#649] [Q&A]
My usecase is that I use MediaCMS as a sort of "self-hosted Youtube", so have a central place where I manage my media-files for various applications.  
So far I've removed all CAN-Things. But I can still use the embed-feature which is great. But whenever I load an embedded frame, it still loads the "Share"-Button in there, even tho I said `CAN_SHARE_MEDIA = False` in my local_settings.py. Any way to remove this or better - that even the embedded frame respects my settings would be great!

**Comment by Husky110** [comment]
As an additive: There might be an even better solution or proposal. :) To get rid of the "share"-button and the channel-icon (so there is no Backlink to the Media-CMS-Instance itself) I had to edit the static/embed.js-file. Really cool would be an alternative-embed-option. Like "embed only the video, without channel-link and without sharing" - or better "dear MediaCMS just give me the plain video-js-implementation and nothing more!". Don't know how to call it... "Silent-Embedding"? Or sth in that manner.

**Reply by german2209** [reply]
Hi,
I'm trying to get rid of the share buttons and backlink also, but can't figure out what I have to change in the embed.js file. 
Can you share with us what you did in your editing?
Regards,
German

**Comment by AzamkhonKh** [comment]
Hello, is there any changes ?

I have same "issue"

**Comment by homegrownhrbs** [comment]
i just added  `button.share-video-btn { display: none !important; }` to the _commons.css  file and it did a great job hiding the share button. 

I would say however, there does need to be a toggle option added to the share > embed menu to disable sharing entirely on embeds for any particular media
